internal/inventory/app/app.go

```diff
@@ -78,7 +77,7 @@ func (app *InventoryApp) Initialize() error {
 
 	app.lc.Info("Starting.")
 
-	if err = app.service.LoadCustomConfig(&app.config, aliasesConfigKey); err != nil {
+	if err = app.service.LoadCustomConfig(&app.config, "AppCustom"); err != nil {
```

Comment:
Please use constant since it is used in line 88 also

Reply:
yep, I have made the change